src/Hud/XpRate/XpRatePlugin.cs

```diff
@@ -108,8 +108,14 @@ public override void Render()
                         if (dif < 0) { bounds.X += dif; bounds.Width -= dif; }
 
                         Graphics.DrawText(timer, Settings.TextSize, new Vector2(bounds.X + 70, position.Y), Settings.TimerTextColor);
-                        Graphics.DrawText(fps, Settings.TextSize, new Vector2(bounds.X + 70, secondLine.Y), Settings.FpsTextColor);
-                        Graphics.DrawText(ping, Settings.TextSize, new Vector2(bounds.X + 70, thirdLine.Y), Settings.LatencyTextColor);
+                        if (Settings.ShowFps)
+                        {
+                            Graphics.DrawText(fps, Settings.TextSize, new Vector2(bounds.X + 70, secondLine.Y), Settings.FpsTextColor);
+                        }
+                        if (Settings.ShowLatency)
```

Comment:
We ask users for ShowLatency only when they have checked "OnlyAreaName" thingy (as shown in the image http://i.imgur.com/rzxz1hq.png). If user haven't checked "Only Area Name", we show them everything by default and there is no option in the menu to enable/disable show latency.

Hence, this if condition is invalid.